apps/server/src/shared/domain/entity/task.entity.ts

```diff
@@ -131,6 +137,21 @@ export class Task extends BaseEntityWithTimestamps implements ILearnroomElement,
 		return finishedIds;
 	}
 
+	public getUsersList(): UsersList[] {
+		const users = this.users.getItems();
+		if (users.length) {
+			const usersList: UsersList[] = users.map((user) => {
+				return {
+					id: user.id,
+					name: `${user.firstName} ${user.lastName}`,
```

Comment:
putting that out like this format might be something we want to talk about with FE/UX right? Just in case somebody thinks it might be an idea to show them like `LASTNAME, FIRSTNAME`

..but that might just be me being overthinking

Reply:
maybe is best for the FE to decide, so let's split this into firstName and lastName